src/platform/Zephyr/CHIPDevicePlatformEvent.h

```diff
@@ -23,7 +23,7 @@
 
 #pragma once
 
-#include <platform/CHIPDeviceEvent.h>
+#include <system/SystemPacketBuffer.h>
```

Comment:
Shouldn't we add it to the Matter upstream also?

Reply:
I think we should, but probably the change is so small and non-controversial that it could be pushed upstream, as a part of 2.6.0 bump.

Reply:
Yes, this was my initial thought as well.